CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Fixed
+- CentOS 7 support by using long sizes only for packets bigger than 4 GiB
```

Comment:
packets -> packages

Could you add some whitespace between headings also?

Reply:
Done.